docs/help-topics/stonesoup-import/stonesoup-import.yml

```diff
@@ -41,16 +41,21 @@
 
-    Before you create and deploy your application, look over the values that display here. You can rename any component from this view, too. In particular, pay attention to the following fields:
+    Before you create and deploy your application, look over the values and options that display here. You can rename any component from this view, too. 
 
-    - **Target port**: Is it correct? If not, click in the port field to modify it.
-    - **CPUs**, **memory**, and **instances**: Choose how many of each of these you want for your application, depending on your subscription.
-    - **Route**: Enter the public URL for your published application.
-    - **Environment variables**: Define your environment variables to customize how we deploy your application. Usually, you define these in a devfile or Dockerfile. If your application doesn’t contain either of these file types, we use an application analyzer to extract the necessary information from your code.
-    - **Runtime**: Did we detect the right runtime? If so, you don’t have to do anything with this field. If you want to change your runtime, click the **Runtime** menu and make a selection.
-    - **Send pull request**: Select this option to create a custom pipeline in your Git repository.
+    - Before you expand all of your configuration options, check the following:
+        - Is the repository correct? Click the GitHub link to check your repository URL or click the repository name above the GitHub link to edit it. When you're satisfied, click the checkmark. To exit editing mode, click the X.
+        - **Runtime**: Did we detect the right one? If so, you don’t have to do anything with this field. If you want to change your runtime, click the runtime name and make a selection.
+    - Expand the section next your repository name, then expand the **Build & deploy configuration** menu to look over these items:
+        - **Dockerfile path**: If we're pointing to the wrong directory for your Dockerfile, click inside this field to make changes.
+        - **Target port**: Is it correct? If not, click in the field to modify it.
+        - **CPU** and **Memory**: Choose how many of each of these you want for your application, depending on your subscription.
+    - Expand **Show advanced deployment options**. 
+        - **Instances**: How many instances of your build image do you want to use?
+        - **Environment variables**: Define your environment variables to customize how we deploy your application. Usually, you define these in a devfile or Dockerfile. If your application doesn’t contain either of these file types, we use an application analyzer to extract the necessary information from your code.
+    - **Build pipeline**: Choose **Default** or **Custom** to specify how to trigger rebuilds.
```

Comment:
Default pipelines are triggered on import, and can be manually triggered from the UI. They also run faster because they only build, skipping some image checks and tests. Custom pipelines are triggered by commits to their source code repo, and run some surface-level checks and security scans, so are more thorough, but take longer because of it.

Reply:
Custom pipelines also have additional requirements, like you need to be the owner of the repo, so that you can install our app in GitHub and grant it access to the source repo. If they want the more advanced pipeline but don't own the repo they're trying to import, they should fork it and start over.

Reply:
whew, tough stuff! how to include these details in mostly positive language? take a look at my changes? 

toward minimalism and clarity, i left out what triggers default pipelines bc i assume it's implied/obvious-ish. (?)

input, please, @mattreid! :)

Reply:
oh, also: begin what again? choosing a pipeline type?

Reply:
Totally! It's not super simple to describe unfortunately :/ 

It can be a good idea to start with Default to ensure RHTAP can build and deploy their component successfully. Once they know it can, they could customize their build pipeline so that the additional security scans and best practice checks are included. Customizing also lets them modify the tasks in the pipeline, which I could see linking to whatever docs we eventually offer on that process later.

If they have to fork, that gives them a different component URL, so they have to go back and change the GitHub URL to the forked repo location.